packages/esm-patient-attachments-app/src/attachments/camera-upload.component.tsx

```diff
@@ -72,12 +72,12 @@ const CameraUpload: React.FC<CameraUploadProps> = ({ onSavePhoto, onTakePhoto, c
           <>
             {!error && <Camera onTakePhoto={handleTakePhoto} onCameraStart={setMediaStream} onCameraError={setError} />}
             <div>
-              <label htmlFor="uploadPhoto" className={styles.choosePhoto}>
-                {t('selectPhoto', 'Select local photo instead')}
+              <label htmlFor="uploadPhotoOrPdf" className={styles.choosePhotoOrPdf}>
+                {t('selectPhotoOrPdf', 'Select local photo or Pdf instead')}
```

Comment:
you also need to reference this change in the [`en.json`](https://github.com/openmrs/openmrs-esm-patient-chart/blob/master/packages/esm-patient-attachments-app/translations/en.json#L12) file